RecoEgamma/PhotonIdentification/data/PHYS14/effAreaPhotons_cone03_pfChargedHadrons_V2.txt

```diff
@@ -0,0 +1,13 @@
+# This file contains Effective Area constants for 
```

Comment:
and here: can be removed

Reply:
You mean removing the effective are files completely?

Reply:
keep only the ones that are used.
I didn't find any use case

Reply:
Ok!